signalfx-org-metrics/README.md.jinja

```diff
@@ -11,20 +11,20 @@
 
 ## DESCRIPTION
 
-SignalFx generates a number of metrics you can use to monitor your usage of SignalFx, including:
+Splunk Infrastructure Monitoring generates a number of metrics you can use to monitor your usage of the application, including:
 
--  Ingest-related metrics that characterize the volume and nature of data that you are sending to SignalFx, such as the number of datapoints that were sent or how many metric time series were created in response to the datapoints and dimensions sent.
+-  Ingest-related metrics that characterize the volume and nature of data that you are sending to Infrastructure Monitoring, such as the number of datapoints that were sent or how many metric time series were created in response to the datapoints and dimensions sent.
 
 -  Usage or engagement metrics, such as the number of dashboards or charts in the organization.
 
--  Metrics that tell you how SignalFx pulls data on your behalf from cloud services like Amazon Web Services or Google Cloud Platform, such as the number of calls to each GCP Stackdriver client method or how many of the calls to the AWS CloudWatch API are being throttled by AWS.
+-  Metrics that tell you how the application pulls data on your behalf from cloud services like Amazon Web Services or Google Cloud Platform, such as the number of calls to each GCP Stackdriver client method or how many of the calls to the AWS CloudWatch API are being throttled by AWS.
 
 Your organization is not charged for these metrics, and they do not count against any limits.
 
 
 ## USAGE
 
-SignalFx admins can see some of these values in built-in charts on the Organization Overview page. And, like any other metrics, you can display them on charts you build to monitor your system health.
+Application admins can see some of these values in built-in charts on the Organization Overview page. And, like any other metrics, you can display them on charts you build to monitor your system health.
```

Comment:
I suggest saying "Admins" instead of "Application admins" here.

Reply:
done